pytorch_lightning/loops/epoch/evaluation_epoch_loop.py

```diff
@@ -250,19 +245,15 @@ def _on_evaluation_batch_start(self, batch: Any, batch_idx: int, dataloader_idx:
         Raises:
             AssertionError: If the number of dataloaders is None (has not yet been set).
         """
-        self.trainer.logger_connector.on_batch_start(batch_idx, batch)
-
-        assert self._num_dataloaders is not None
-        self.trainer.logger_connector.on_evaluation_batch_start(dataloader_idx, self._num_dataloaders)
```

Comment:
where did on_evaluation_batch_start go?

Reply:
I guess it used to attach dataloader_idx to the model which is no longer the case now since `results` controls it now.

Reply:
We don't need it anymore, the only thing it did was set the `dataloader_idx` in the model.

It gets set now in `LoggerConnector.on_batch_start`